src/omero_zarr/masks.py

```diff
@@ -45,201 +45,308 @@ def image_masks_to_zarr(image, args):
 
     dtype = MASK_DTYPE_SIZE[int(args.mask_bits)]
 
+    if args.style == "labelled" and args.mask_bits == "1":
```

Comment:
Can you change the default for `mask_bits` so it's automatic depending on whether this is `labelled` or not?

Reply:
You mean leave the default as `None` and describe the behavior in the docs?